package.json

```
@@ -1,13 +1,13 @@
 {
   "name": "sundial",
-  "version": "1.6.0",
+  "version": "1.7.1",
```

Reporting changelog version 
```suggestion
  "version": "0.1.0",
```